tools/api-markdown-documenter/src/test/snapshots/html/simple-suite-test/default-config/index.html

```diff
@@ -1,43 +1,28 @@
-<!DOCTYPE html>
+<!doctype html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
+    <meta charset="utf-8">
   </head>
   <body>
     <section>
-      <h1>
-        API Overview
-      </h1>
+      <h1>API Overview</h1>
       <section>
-        <h2>
-          Packages
-        </h2>
+        <h2>Packages</h2>
         <table>
           <thead>
             <tr>
-              <th>
-                Package
-              </th>
-              <th>
-                Description
-              </th>
+              <th>Package</th>
+              <th>Description</th>
             </tr>
           </thead>
           <tbody>
             <tr>
-              <td>
-                <a href='./test-suite-a'>test-suite-a</a>
-              </td>
-              <td>
-                <b>Test package</b>
-              </td>
+              <td><a href="./test-suite-a">test-suite-a</a></td>
+              <td>&#x3C;b>Test package&#x3C;/b></td>
```

Comment:
Is this correct? Seems to have mangled the `<` for the `<b>` elements.

Reply:
Oh, good catch. Yeah, this is incorrect. The code is incorrectly escaping HTML that came from the code comment itself (which is supposed to be supported). I'll add a unit test for this case as well. This is important.

Reply:
Okay, so after digging into this for a while, I have formed an opinion on something I think I will change (in a separate PR). The logic that was broken here was our use of unsafe HTML. TSDoc parses HTML tags and passes them to us as "escaped text", with the expectation that they will be forwarded to the renderer untouched. This is somewhat sketchy to begin with.

Secondly, the tree structure they give is bananas. For the comment text `<b>Hello world!</b>`, the tree TSDoc generates is:
```json
[
    { type: "HtmlStartTag", "tag": "b" }
    { type: "EscapedText", text: "Hello world!" },
    { type: "HtmlCloseTag", "tag": "b"}
]
```

rather than something like:

```json
{
    "type": "HtmlBlock",
    "tag": "b",
    "children": [
        { "type": "text", "text": "Hello world!" }
    ]
}
```

This makes it difficult to preserve intention when mapping to our output domains. We got around this by treating all of the adjacent nodes as "escaped" and letting the contents get mapped to output raw. Sketchy, but it's what the base `api-documenter` library does.

I double checked, and intellisense simply ignores HTML tags in comments. It renders the inner contents, but completely ignores any tags decorating them.

My opinion: do this. It's safer, better aligns with TSDoc spec (which makes no claims about support for embedded HTML AFAIK), and makes our transformation logic A LOT simpler.

I'll put up a separate PR that makes this change.

Reply:
PR: https://github.com/microsoft/FluidFramework/pull/22671